applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -98,6 +98,8 @@ For example:
 
 For information about how to upgrade the device firmware using a PC or a smartphone, see the :ref:`matter_bridge_app_dfu` section.
 
+At that moment DFU over BT_SMP in unavaliable on nrf54h20 board.
```

Comment:
You can use the "note" format to make this information better visible on the page:
```suggestion
.. note::
    Currently the DFU over Bluetooth LE SMP and Matter OTA are not available for the nrf54h20 board target.
```